### PR TITLE
Allow plot MMM components in the original scale

### PR DESCRIPTION
**Pull request by juanitorduz**
We can not get the plot in the example notabook 

image

----
📚 Documentation preview 📚: https://pymc-marketing--870.org.readthedocs.build/en/870/

### CODE REVIEW

File: pymc_marketing/mmm/delayed_saturated_mmm.py [hunk 1]
```diff
@@ -1072,6 +1072,166 @@ def plot_channel_parameter(self, param_name: str, **plt_kwargs: Any) -> plt.Figu
...
+        if original_scale:
+            return apply_sklearn_transformer_across_dim(
+                data=contributions,
+                func=self.get_target_transformer().inverse_transform,
+                dim_name="date",
+            )
+
+        return contributions
+
+    def plot_components_contributions(
```
Comment (juanitorduz) [inline pymc_marketing/mmm/delayed_saturated_mmm.py:1104]: This overwrites the method from the base class as in the MMM class we are assuming we have scalers.

File: pymc_marketing/mmm/delayed_saturated_mmm.py [hunk 2]
```diff
@@ -1072,6 +1072,166 @@ def plot_channel_parameter(self, param_name: str, **plt_kwargs: Any) -> plt.Figu
...
 
+    def get_ts_contribution_posterior(
+        self, var_contribution: str, original_scale: bool = False
+    ) -> DataArray:
+        """Get the posterior distribution of the time series contributions of a given variable.
+
+        Parameters
+        ----------
+        var_contribution : str
+            The variable for which to get the contributions.
```
Comment (wd60622) [inline pymc_marketing/mmm/delayed_saturated_mmm.py:1083]: Does this work for all variables? Or is there only a few that work? Maybe list out

**Review by wd60622 — APPROVED** [review]
Looks good. Just comment on doc

File: pymc_marketing/mmm/delayed_saturated_mmm.py (hunk 2 above)
Comment (juanitorduz) [reply pymc_marketing/mmm/delayed_saturated_mmm.py]: Added better description in the doctring 👍

**Pull request merged**